examples/map-proof.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Example how to use `MapProof`s in client apps.
+ */
+'use strict'
+
+const exonum = require('..')
```

Comment:
Very-very minor nitpick, could you please write it with capital `E` to match overall design:
```javascript
const Exonum = require('..')
```